src/utils.rs

```diff
@@ -383,7 +383,7 @@ pub(crate) fn float_repr<F: Float>(f: F) -> alloc::string::String {
                 if i == 0 { "" } else { ":" }
             );
         }
-        out += &format!(" ({:.10E})", f);
+        out += &format!(" ({f:.10E})");
```

Comment:
My only comment would be these formatting changes may change the MSRV (beyond that of nalgebra update if it increases it as well). I'm not the main person on this repo so I'll leave it to @vadixidav to decide

Reply:
Oh yeah, good point. I just blindly obeyed clippy's bidding...

Reply:
It may be fine, I'd hold off on reverting until someone else pitches in - plus it might be a lower MSRV than nalgebra anyway

Reply:
Will do :)